Packs/GitHub/Integrations/GitHub/GitHub_test.py

```diff
@@ -396,3 +395,35 @@ def test_list_issue_comments_since(mocker):
     list_issue_comments(issue_number, since_date)
     request_args = patched_request.call_args
     assert 'since' in request_args.kwargs['params']
+
+
+@pytest.mark.parametrize('args, response_content, expected_result', [
+    ({'assignee': 'user1', 'pull_request_number': '1'}, b'{"message": "success"}',
+     "The following users were assigned successfully to PR #1: \n['user1']"),
+    ({'assignee': 'user1, user2', 'pull_request_number': '1'}, b'{"message": "success"}',
+     "The following users were assigned successfully to PR #1: \n['user1', 'user2']"),
+    ({'assignee': 'user3', 'pull_request_number': '1'}, b'{"message": "success"}',
+     "\nThe following users were not assigned to #1: \n['user3'] \nVerify that the users exist and that you have the "
+     "right permissions."),
+])
+def test_assignee(mocker, args, response_content, expected_result):
+    """
+    Given:
+      - Case 1: one valid user to be assigned to PR
+      - Case 2: two valid users to be assigned to a PR
+      - Case 3: one not valid user to be assigned to a PR
+    When:
+      - all the args are given
+    Then:
+      - Ensure the correct behaviour
+      - Case 1: Should return Success for one provided user
+      - Case 2: Should return Success for two provided users
+      - Case 3: Should return that assignment failed for provided user
+    """
+    mocker.patch.object(demisto, 'args', return_value=args)
+    GitHub.USER_SUFFIX = '/repos/user/repo'
+    mocker.patch('GitHub.http_request', return_value={"assignees": [{"login": "user1"}, {"login": "user2"}]})
```

Comment:
This will work but not 100% professionally (it works because the response doesn't include usr3 but it does have 3 unrelated user). My suggestion is to do the same thing you did at the beginning - add a nested function:`def http_request_mock(method=None, url_suffix=None, data=post_data)` it will mimic the Github service. inside the function check the post data, if the users there also appear in the approved list (which is only usr1 and usr2) return them as assignees else return an empty list of assignees.

```suggestion
    mocker.patch('GitHub.http_request', return_value={"assignees": [{"login": "user1"}, {"login": "user2"}]})
```